modules/exploits/unix/http/pfsense_config_data_exec.rb

```diff
@@ -0,0 +1,190 @@
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::FileDropper
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'pfSense Restore RRD Data Command Injection',
+        'Description' => %q{
+          This module exploits an OS Command Injection vulnerability in the pfSense
+          Config Module (CVE-2023-27253). The vulnerability affects versions <= 2.6.0
+          and can be exploited by an authenticated user if they have the
+          "WebCfg - Diagnostics: Backup & Restore" privilege.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Emir Polat', # vulnerability discovery & metasploit module
+        ],
+        'References' => [
+          ['CVE', '2023-27253'],
+          ['URL', 'https://redmine.pfsense.org/issues/13935']
```

Comment:
https://github.com/pfsense/pfsense/commit/ca80d18493f8f91b21933ebd6b714215ae1e5e94 should also be added in here as a reference link for the commit that fixed this.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/17861/commits/3560572130cf9ece7ab65dad460db45ff4903d2e

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/commit/3560572130cf9ece7ab65dad460db45ff4903d2e